HandsOn/Group04/mappings/mappings-with-links-updated.yml

```diff
@@ -0,0 +1,102 @@
+prefixes:
+    rr: http://www.w3.org/ns/r2rml#
+    foaf: http://xmlns.com/foaf/0.1/
+    xsd: http://www.w3.org/2001/XMLSchema#
+    rdfs: http://www.w3.org/2000/01/rdf-schema#
+    dc: http://purl.org/dc/elements/1.1/
+    rev: http://purl.org/stuff/rev#
+    gtfs: http://vocab.gtfs.org/terms#
+    geo: http://www.w3.org/2003/01/geo/wgs84_pos#
+    owl: http://www.w3.org/2002/07/owl#
+    schema: http://schema.org/
+    dct: http://purl.org/dc/terms/
+    rml: http://semweb.mmlab.be/ns/rml#
+    ql: http://semweb.mmlab.be/ns/ql#
+    pproc: http://contsem.unizar.es/def/sector-publico/pproc#
+    rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+    dp: http://dbpedia.org/ontology/
+    organ: http://www.w3.org/ns/org#
+    pp: http://exercise.publicprocurement.es/about/ontology/group04pp/
+    
+mappings:
+  Department:
+    sources:
+      - [purchase-order-2014-2015-updated-with-links.csv~csv]
+    s: dp:Department#$(Department Name)
```

Comment:
Normally the IRIs will be slash IRIs and not has IRIs

Reply:
Sorry, I'm not particularly sure what you mean. I know I seem to have missed a slash between prefix and department, is that the correct one?

prefix:
...
dp: http://dbpedia.org/ontology
...

s: dp:/Department#$(Department Name)

Reply:
s: dp:Department/$(Department Name)